src/Symfony/Component/Asset/EventListener/PreloadListener.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Asset\EventListener;
+
+use Symfony\Component\Asset\Preload\PreloadManager;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+/**
+ * Adds the preload Link HTTP header to the response.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class PreloadListener implements EventSubscriberInterface
+{
+    private $preloadManager;
+
+    public function __construct(PreloadManager $preloadManager)
+    {
+        $this->preloadManager = $preloadManager;
+    }
+
+    public function onKernelResponse(FilterResponseEvent $event)
+    {
+        if (!$event->isMasterRequest()) {
+            return;
+        }
+
+        if ($value = $this->preloadManager->buildLinkValue()) {
+            $event->getResponse()->headers->set('Link', $value);
```

Comment:
you must pass ``false`` as third argument. You don't want to replace the existing Link headers which might exist for other purposes